org.scala-ide.sdt.core/src/org/scalaide/core/IScalaProject.scala

```diff
@@ -151,7 +153,7 @@ trait IScalaProject extends Publisher[IScalaProjectEvent] {
    * Initialization for the build manager associated to this project
    * @return an initialized EclipseBuildManager
    */
-  def buildManager(): EclipseBuildManager
+  def buildManager(): BuildScopeUnitsManager
```

Comment:
Nor binary compatible. We need to keep compatibility, so either we make the new manager a subtype of the previous one, or deprecate this method and add another one here.